spec/card_spec.rb

```diff
@@ -44,5 +45,11 @@
         expect(card.right_translation?("Hello")).to eq(false)
       end
     end
+
+    describe "user" do
+      it "validates that the card belongs to a user" do
+        expect(card.user.name).to eq("kenn")
+      end
+    end
```

Comment:
maybe i should only test methods that are in the Model? or if it's related to user input?